datajunction-server/datajunction_server/api/cubes.py

```diff
@@ -48,6 +51,24 @@
 router = SecureAPIRouter(tags=["cubes"])
 
 
+@router.get("/cubes", name="Get all Cubes")
+async def get_all_cubes(
+    *,
+    session: AsyncSession = Depends(get_session),
+    catalog: Optional[str] = Query(
```

Comment:
Might be fine for now, but would be nicer if we can add pagination in here too, since with cube revision metadata for all cubes, this payload could become quite large.

Reply:
Good call! I'll make a follow-up PR and I added issue #1392 to not lose track.